ntc_templates/templates/juniper_junos_show_system_configuration_database_usage.textfsm

```diff
@@ -0,0 +1,17 @@
+Value MAX_DB_SIZE (\S+)
+Value CURR_DB_SIZE (\S+)
+Value ACTUAL_DB_SIZE (\S+)
+Value AVAIL_DB_SPACE (\S+)
+
+# If the database is being modified, a "cannot proceed" message will be displayed
+# Otherwise, the database usage will be displayed
+
+Start 
+  ^\s*Maximum\s+size\s+of\s+the\s+database:\s+${MAX_DB_SIZE}\s*
+  ^\s*Current\s+database\s+size\s+on\s+disk:\s+${CURR_DB_SIZE}\s*
+  ^\s*Actual\s+database\s+usage:\s+${ACTUAL_DB_SIZE}\s*
+  ^\s*Available\s+database\s+space:\s+${AVAIL_DB_SPACE}\s* -> Record
+  ^{(master|backup)}\s*$$
+  ^\s*Cannot\s+proceed,\s+database\s+is\s+currently\s+being\s+modified -> Error
```

Comment:
@jnicholson56
Would you by chance add raw output to this PR for an instance where "cannot proceed" is present?

Please and thank you!

Reply:
@jnicholson56 
In hindsight this may not work out well for the unit tests since this triggers the Error state.

As a suggestion, please test on your local machine with `invoke` first to make sure it's kosher before committing/pushing the changes up to GitHub. _(This way you should be able to save yourself the hassle of having to revert the changes if this turns out to be a bad idea.)_

* https://github.com/networktocode/ntc-templates/blob/master/docs/dev/dev_parser.md#testing
* https://github.com/networktocode/ntc-templates/blob/master/docs/dev/dev_environment.md#invoke

Reply:
> Yes I can do that. It might take me a day or two to stage the conditions
required.

@jnicholson56
I guess we'll let this one ride as it is.

**Question:** :question:
I suppose the idea was to rather have the output not parse because it throws an error as opposed to parse and contain empty strings for each key?

Reply:
Here's my suggestion so parsing doesn't throw an error -- though it still results in an empty list which can be easily tested for. _Best of both worlds?_

```suggestion
  ^\s*Cannot\s+proceed,\s+database\s+is\s+currently\s+being\s+modified
```